frontend/app-development/queries/mutations.ts

```diff
@@ -35,3 +39,26 @@ export const deleteLanguageCode = (org, app, language) =>
 
 export const upsertTextResources = (org, app, language, payload) =>
   put(textResourcesPath(org, app, language), payload);
+
+export const saveFormLayout = (org, app, layoutName, payload) =>
```

Comment:
Hva tenker du om at vi legger til typer for argumentene i funksjonene? :)

Reply:
Enig! Separat oppgave?